### PR TITLE
setup script to run both servers at same time

### PR DESCRIPTION
**Pull request by Dan-Y-Ko**
use "yarn dev" or "npm run dev" to boot up both servers at the same time

### CODE REVIEW

**Comment by ArnasDickus** [comment]
Do you know why Travis test doesn't pass?

**Comment by Dan-Y-Ko** [comment]
probably because i changed the folder structure a bit

**Comment by mubarakshow** [comment]
@Dan-Y-Ko

**Status**: rejected
**Reasons**: 
1. your PR didn't pass Travis tests and that's because the build script is missing from the package.json file. 

2. according to @ArnasDickus in react npm run build compiles the project and when we had separated package.json files everything was working fine. But with this update @Dan-Y-Ko merged package.json file into one but didn't added build script for some reason.

**Fix**: i suggest you add a build script to the package.json file.

File: package.json [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-    "react-map-gl": "^4.0.11",
-    "react-mapbox-gl": "^4.2.0",
-    "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.5",
-    "styled-components": "^4.1.3",
-    "tachyons": "^4.11.1"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
```
Comment (mubarakshow) [inline package.json:19]: @Dan-Y-Ko this was the build script before you made changes

File: package.json [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ]
+    "name": "mappypals",
+    "version": "1.0.0",
+    "description": "track your friend abroad",
+    "main": "server.js",
+    "scripts": {
```
Comment (mubarakshow) [inline package.json:6]: the build script is missing here @Dan-Y-Ko @ArnasDickus

**Pull request closed without merging**